### DISCUSSION TITLE
I would like to contrebute to this project!

### DISCUSSION THREAD

**Discussion by ice19942335** [dotnet/aspnetcore#22384] [General] [answered]
Hello. 

I am a junior developer. Main stack C#, JS, HTML/CSS

I would like to contribute to open source project, I like ASP.Net Core because it is amazing framework!

Is it there any job for me please?
Make documentation will be fine as well :)

Kind regards
Alex

**Comment by mkArtakMSFT** [comment]
Thanks for your willingness to help and learn!
You can scan through the issues, which have the `help wanted` label and choose the one you want to work on. here is the query: https://github.com/dotnet/aspnetcore/issues?q=is%3Aopen+is%3Aissue+milestone%3ABacklog+label%3A%22help+wanted%22

**Comment by Tratcher** [comment, marked as answer]
See https://github.com/dotnet/aspnetcore/blob/master/CONTRIBUTING.md
